src/collections/_documentation/workflow/releases/index.md

```diff
@@ -334,3 +334,74 @@ You can also use our [API]({%- link _documentation/api/releases/post-release-dep
+
+[ SCREENSHOT: Releases index page ]
+
+### Release Details
+
+The release details page focuses on an individual release. Elements of the release are presented in visual trends for crashes and sessions, specifics regarding each issue, and commit author breakdowns.
+
+[ SCREENSHOT: Releases detail page ]
+
+For more information about Sentry's Mobile features, see the full documentation for [Android SDK]({%- link _documentation/platforms/android/index.md -%}) and [iOS SDK](docs.sentry.io/platforms/cocoa/).
+
+### Session Data
+
+A [session]({%- link _documentation/workflow/releases/release-health-glossary.md -%}#session) begins the moment a user starts interacting with an application or when an application resumes from the background into the foreground. With Release health, you can track the percentage of users using sessions.
+
+For more details, see the [full documentation on session payloads]({%- link _documentation/development/sdk-dev/event-payloads/session-fields.md -%}).
+
+### Data Filtering
+
+Sentry has inbound data filters. These filters are exclusively applied at ingest time and not later in processing. This, for instance, lets one discard an error by error message when the error is ingested through the JSON API. On the other hand, this filter doesn't apply to ingested minidumps.
+
+Data filters are only partially supported for session events, and they might be working slightly different:
+
+#### Equivalent Filters
+
+**release filter**
+
+This filter discards events of a particular release. If sessions are started or updated and would match this filter, sessions are silently discarded entirely. This is consistent with error events.
+
+**web crawler filter**
+
+When session events send their user agent, they are discarded the same way as error events are. There might be a slight difference in the fallback case where the user agent of the sending SDK is used (browser javascript).
+
+**legacy browsers**
+
+This works exactly like the web crawler filter. No sessions are created or updated for legacy browsers when enabled.
+
+**ip filter**
+
+Throws away sessions of a specific IP.
+
+#### Unsupported Filters
+
+**error message filter**
+
+This filter does not apply for sessions. Even if a user is discarding such errors, they will count against their stability score. An alternative measure *could be* to discard such sessions entirely if an error of this type is filtered out, but we believe this would make the system harder to understand.
+
+**browser extensions**
+
+Since the logic for detecting such errors requires all the error information, we cannot support this filter at the session ingest time. If we want to improve this case, we could retroactively discard such sessions entirely once we identify such an error case.
+
+### FAQ
+
+1. When you change the time range, the crash-free settings and release adoption will be updated, and Sentry will use the new time range.
```

Comment:
I would add here:
Sentry doesn't provide average value of the daily crash free users or daily crash free sessions but the real value of those metrics calculated for the selected time period.
The number can significantly change based on the selected time range. For example:
If you have 7 users that are using your application for 7 days and each day one of them will experience a crash. Release Index Page with the 24h time range selected will show you every day 85,7% Crash Free Users on the release. If each day different user experience so by the end of the week when you change the time range in the release index view to 7 days, the crash free users on the project will be 0%.